### PR TITLE
First pass of CSP-compliance for mousetrap

### PR DESCRIPTION
**Pull request by jvoisin**
Since the preference for the keyboard shortcuts
is set server-side, I had to shove them into a non-javascript
context, and then access them via javascript,
in a separate file.

Since I'm not a javascript expert, I'm more than open
to alternatives if this isn't the right way™ to do it.

### CODE REVIEW

**Comment by jvoisin** [comment]
This is related to #909

File: airsonic-main/src/main/webapp/WEB-INF/jsp/head.jsp [hunk 1]
```diff
@@ -19,39 +19,10 @@
...
-    Mousetrap.bind('g a', function() { parent.frames.main.location.href = "help.view?"; });
-    Mousetrap.bind('?',   function() { parent.frames.main.location.href = "more.view#shortcuts"; });
-}
+<script id="preferencesConfig" type="application/x-configuration">
+  {
+    "keyboardShortcutsEnabled": ${model.keyboardShortcutsEnabled ? 'true' : 'false'}
+  }
 </script>
+<script defer type="text/javascript" src="<c:url value="/script/mousetrap-1.6.0.js"/>"></script>
+<script defer type="text/javascript" src="<c:url value="/script/keyboards_shortcuts.js"/>"></script>
```
Comment (fxthomas) [inline airsonic-main/src/main/webapp/WEB-INF/jsp/head.jsp]: Typo in the filename (`keyboards` → `keyboard`)

**Comment by jvoisin** [comment]
Err, I forgot to push my local branch :/
I've squash'n'rebased everything, tested it locally, it's now working, sorry about that.

**Comment by fxthomas** [comment]
I had forgotten about this, but it's been running without issues for me for a few weeks now.

**Pull request merged**